libs/eo/emissions/shell/src/lib/eo-emissions-page-info.component.ts

```diff
@@ -0,0 +1,72 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ChangeDetectionStrategy, Component, NgModule } from '@angular/core';
+import { MatCardModule } from '@angular/material/card';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  selector: 'eo-emissions-page-info',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+      mat-card {
+        background: var(--watt-color-state-warning-light);
+        h3 {
+          font-weight: 600; // Magic number by designer
+        }
+
+        .coming-soon-overlay {
+          background-color: rgba(196, 196, 196, 0.7);
+          width: 100%;
+          height: 100%;
+          position: absolute;
```

Comment:
So um... the... design has changed 😄  haha... we need to change this again. It is now a smaller font and in the top right corner